addon/components/bs-form-element.js

```diff
@@ -270,7 +270,7 @@ export default FormGroup.extend({
    * @property model
    * @public
    */
-  model: computed.alias('form.model'),
+  model: null,
```

Comment:
It should be sufficient to just declare the property as `computed.reads('form.model')` (one way binding), without the additional code in `init()`, or do I miss something? This way it would default to `form.model`, but would allow to be overridden without changing the form`s `model`. That's what you are trying to achieve, aren't you?